docker-compose.yml

```diff
@@ -15,4 +15,23 @@ services:
           - AIRFLOW__SMTP__SMTP_MAIL_FROM=${AIRFLOW__SMTP__SMTP_MAIL_FROM}
           - AIRFLOW__SMTP__SMTP_SSL=${AIRFLOW__SMTP__SMTP_SSL}
           - AIRFLOW__SMTP__SMTP_PORT=${AIRFLOW__SMTP__SMTP_PORT}
+          - POSTGRES_USER=postgres_user
+          - POSTGRES_PASSWORD=postgres_pass
+          - POSTGRES_HOST=postgres
+          - POSTGRES_PORT=5432
+          - POSTGRES_DB=opac_airflow
```

Comment:
No composer de `production ready` nós precisamos manter essas variáveis de ambiente com dados fixados ou deveríamos deixa-las sem dados para forçar a configuração?